package.json

```diff
@@ -59,6 +59,7 @@
     "test:eslint": "eslint --color src",
     "test:jest": "jest --colors",
     "test:prettier": "prettier -l \"src/**/*\"",
+    "test:attw": "attw . --pack --profile node16",
```

Comment:
I believe we do not need to support `node10` anymore. Otherwise we would need to create a dedicated type file and add it to the root, for the `compat` export.